cucoslib/workers/gh_metadata_init.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from cucoslib.base import BaseTask
+from cucoslib.models import Analysis
+
+
+class GitHubManifestMetadataInit(BaseTask):
+    def execute(self, arguments):
+        self._strict_assert(arguments.get('git_repo_url'))
```

Comment:
Not yet, but https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/27/files#diff-8ccf2d5f54c7573302ac2b0200a732edR255 will introduce it. Let's be consistent as we might want to do monitoring on these projects as well. So either I will introduce `git_repo_url` or you make the change. It's up to you :)